.editorconfig

```diff
@@ -38,6 +38,9 @@ indent_style = tab
 charset = utf-8
 insert_final_newline = true
 
+[Python.Runtime.csproj]
+charset = latin1
```

Comment:
This is an odd choice for XML, is this intentional?

Reply:
Without this Visual Studio tends to add UTF-8 BOM header to solution, leading to PRs with the entire file replaced. It simply ignores `utf-8`.